src/core/GOBitmap.h

```diff
@@ -11,30 +12,31 @@
 
 class wxImage;
 
-class GOBitmap
-{
-private:
-	wxImage* m_img;
-	wxBitmap m_bmp;
-	double m_Scale;
-	int m_ResultWidth;
-	int m_ResultHeight;
-	unsigned m_ResultXOffset;
-	unsigned m_ResultYOffset;
+class GOBitmap {
+ private:
```

Comment:
Something went wrong.

When I run clang-format, I receive

```
class GOBitmap {
private:
  wxImage *m_img;
```

Could you paste the result of ``clang-format --dump-config | grep AccessModifierOffset`` here?

Reply:
@oleg68 You are on my branch ?

Reply:
You have the wrong format locally ?
Because the pre-commit build here on github passes (so, the clang-format output is the same as mine).

Reply:
![image](https://user-images.githubusercontent.com/19529533/148654056-fe20e3bc-59a6-4c72-9562-864a9f1be62f.png)